examples/extended/parallel/MPI/source/src/G4MPIhistoMerger.cc

```diff
@@ -52,8 +52,9 @@ void G4MPIhistoMerger::Merge()
     G4cout << "Starting merging of histograms" << G4endl;
   }
 
-  const MPI::Intracomm* parentComm = G4MPImanager::GetManager()->GetComm();
-  MPI::Intracomm comm = parentComm->Dup();
+  const MPI_Comm* parentComm = G4MPImanager::GetManager()->GetComm();
+  MPI_Comm comm;// = parentComm->Dup();
```

Comment:
```suggestion
  MPI_Comm comm;
```
No need for comment.